models/brain_models.py

```diff
@@ -0,0 +1,33 @@
+from candidate_models.model_commitments import cornet_brain_pool
+from models import test_models
+
+"""
+Template module for a brain model submission to brain-score
+"""
+
+
+def get_model_list():
+    """
+    This method defines all submitted model names. It returns a list of model names.
+    The name is then used in the get_model method to fetch the actual model instance.
+    If the submission contains only one model, return a one item list.
+    :return: a list of model string names
+    """
+    return cornet_brain_pool.keys()
```

Comment:
as above, the method suggest a `list` should be returned

Reply:
Yes, a list is returned?

Reply:
Ah this must have changed recently -- at least in earlier python versions, `dict.keys()` would return an iterable keys view that could only be iterated over once. Seems like that is not the case anymore, so all good.